.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: "\U0001F41EBug report"
+about: Report a bug in webhint
+---
+<!--
+
+Hi there 👋
+
+Thanks for taking the time to make webhint better! Before
+opening a new issue please make sure to search in the existing
+ones (even closed!) as they might contain information about
+workarounds, resolution, or progress updates.
+
+-->
+
+# 🐛 Bug report
+
+## Description
+
+<!-- ✍️ What were you doing? How do we repro your issue? -->
+
+## Environment
+
+* __`hint` version:__ <!-- ✍️ Run `hint -v` -->
+* __`Node.js` version:__ <!-- ✍️ Run `node -v` -->
+* __`OS`:__ <!-- ✍️ -->
+* __URL for which webhint failed:__ <!-- ✍️ -->
```

Comment:
Most of these pieces of environment data will only apply for the CLI (as will other details such as running `--debug` further down). When a user is reporting a bug in the online scanner, VS Code extension, or the browser extension we'll probably want to ask for slightly different information. 

I could easily see having separate "report a bug" templates for each of these targets (though we don't necessarily need to add all of them right now). Thoughts?

Reply:
Yes, we can probably have separate templates for those and add them once they are more broadly used.